src/pages/Home/components/ModalWindow/MovieAdd/MovieAdd.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import ModalWindow from 'pages/Home/components/ModalWindow/ModalWindow';
+import 'pages/Home/components/ModalWindow/MovieAdd/MovieAdd.scss';
+
+class MovieAdd extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            id: '',
+            title: '',
+            date: '',
+            genre: '',
+            overview: '',
+            runtime: ''
+        };
+
+        this.handleMovieAddIdChange = this.handleMovieAddIdChange.bind(this);
+        this.handleMovieAddTitleChange = this.handleMovieAddTitleChange.bind(this);
+        this.handleMovieAddDateChange = this.handleMovieAddDateChange.bind(this);
+        this.handleMovieAddGenreChange = this.handleMovieAddGenreChange.bind(this);
+        this.handleMovieAddOverviewChange = this.handleMovieAddOverviewChange.bind(this);
+        this.handleMovieAddRuntimeChange = this.handleMovieAddRuntimeChange.bind(this);
+        this.handleMovieAddSubmit = this.handleMovieAddSubmit.bind(this);
+    }
+
+    handleMovieAddIdChange(event) {
+        this.setState({
+            id: event.target.value
+        });
+    }
+
+    handleMovieAddTitleChange(event) {
+        this.setState({
+            title: event.target.value
+        });
+    }
+
+    handleMovieAddDateChange(event) {
+        this.setState({
+            date: event.target.value
+        });
+    }
+
+    handleMovieAddGenreChange(event) {
+        this.setState({
+            genre: event.target.value
+        });
+    }
+
+    handleMovieAddOverviewChange(event) {
+        this.setState({
+            overview: event.target.value
+        });
+    }
+    handleMovieAddRuntimeChange(event) {
+        this.setState({
+            runtime: event.target.value
+        });
+    }
```

Comment:
hey, @nmogylchuk I would like to suggest to search some info on how to avoid `.bind` in react components.

another thing I wanted to mention is that you can give every input `name` property and thus all these handlers could be rewritten into one:

```
handleChange = event => {
   const { name, value } = event.target;
   this.setState({
      [name]: value
   });
}
``` 
what do you think about it?